sos/report/plugins/hbn.py

```diff
@@ -12,13 +12,30 @@
 
 class HBN(Plugin, IndependentPlugin):
 
-    short_desc = 'HBN (Host Based Network)'
+    short_desc = 'HBN (Host Based Networking)'
     plugin_name = "hbn"
+    vsctl = "ovs-vsctl"
```

Comment:
what is the package that install it?

Reply:
openvswitch.
In some OSes, it is also openvswitch or openvswitch-switch. 
But multiple packages can install, for doca, it is doca-openvswitch. 
I will update.

Reply:
Since the ovs output is collected by the ovs plugin, I removed this dependency.

Reply:
is `vsctl` used?